spec/usage/components/q_spec.rb

```diff
@@ -0,0 +1,44 @@
+require_relative "../../support/utils"
+include Utils
+
+describe "Q Component", type: :feature, js: true do
+
+  it "Example 1" do
+
+    class ExamplePage < Matestack::Ui::Page
+
+      def response
+        components {
+          #simple q
+          q
+
+          #simple q with attributes
+          q id: "my-id", class: "my-class"
+
+          #nested q
+          q do
+            q id: "my-nested-q"
```

Comment:
Please test the `text: "This is an example text cite"` and `cite: "www.matestack.org/example"` as well :)